README.md

```diff
@@ -1,65 +1,133 @@
-2. Create a configuration file by following the instruction in the configuration section below.
+1. Compile the jar file: `mvn clean package [-DskipITs=true]`
 
-If the manual compilation was successful, you should see a JAR file in the `target` directory, with a name like `data-exporter-VERSION_NUMBER-jar-with-dependencies.jar`. This is the file you will run with a command like the following:
+The option "-DskipITs=true" is optional and allows for running only of unit tests and not integration tests, if 
+desired, since integration tests require actual connection parameters be present in a test configuration file added
+by the user at the file path "src/test/resources/real_config.properties" (see [Integration Tests](#integration-tests)
+section for more information).
 
-4. `java -jar target/data-exporter-1.0-SNAPSHOT-jar-with-dependencies.jar [path_to_config_file]`
+If the manual compilation was successful, you should see a JAR file in the `target` directory, with a name like 
+`data-exporter-VERSION_NUMBER-jar-with-dependencies.jar`. This is the file you will run with a command like the 
+following (the `-c|--overwrite_config_file flag is optional and forces a new configuration file to be created through
+prompts from the user for new values).
 
-## Configuration
+2. `java -jar target/data-exporter-1.0-SNAPSHOT-jar-with-dependencies.jar [-c|--overwrite_config_file]`
 
-**NOTE: Configuration is done when calling the main script `runDataExporter.sh` for the first time, so this section is if set values need to be changed and for general reference.**
+## Configuration
 
-A auto-configuration script is provided at the root directory of this project and is called `configureDataExporter.sh`.  On first running the Icon Finder tool the configuration script will be run automatically and create a configuration file.   If the configuration needs to be changed, however, this script can be run directly: `./configureDataExporter.sh`.
+**NOTE: Configuration is done when calling the main script `runDataExporter.sh` for the first time (via invocation of
+the `data-exporter.jar`), so this section is if set values need to be changed as well as for general reference.**
 
-The configuration file produced by the script will be at the root directory of this project and named `config.properties`.  It can be viewed and edited directly if desired.
+The configuration file produced will be at the root directory of this project and named `config.properties`.  It can 
+be viewed and edited directly if desired.
 
-A sample configuration file is provided at `src/main/resources/config.properties` and looks like this, but should **NEVER BE EDITED DIRECTLY** and any changes are ignored by git after running the configuration script.
+A sample configuration file is provided at `src/main/resources/config.properties` and looks like this, but should 
+**NEVER BE EDITED DIRECTLY** and any changes are ignored by git after running the configuration script.
```

Comment:
> NEVER BE EDITED DIRECTLY...
Why not? If a user knows all of the configuration values, why can't they edit the file directly?

Reply:
The file path was actually a typo - it should be for the file name `sample_config.properties`.

This file shouldn't be edited because it is only a template and the actual configuration created and used is `config.properties`.  The warning not to edit the sample file directly is an extra measure of precaution to prevent sensitive information from being committed (the other measure being the `git update-index --assume-unchanged ...` command, but this is only run if the shell script is run).  

Also, there is no need to change the contents of the sample configuration file because it is only for reference of what key/value pairs are needed by the code.

I've included some clarification in the README: https://github.com/reactome/release-data-exporter/pull/4/commits/3924632bde7f6db33379c6abf8a0684023076ab8